Documentation/adding-new-repository.md

```diff
@@ -0,0 +1,111 @@
+# How to add a new repository into the Mender ecosystem
+
+Adding a new repository requires changes in several places. This document aims to be a full recipe of all components and steps that require modifications.
+
+It is recommended to take a repository in the same domain (backend, client, etc.) as a model when going through the different phases.
+
+## GitHub: create repo and give access
+
+Create the new repository. Consult with the team, epic owner or product owner if the repository is to be open or private
+
+By default, we give write access for `developers` and `rndity` groups. Remove any individual user.
+
+ref https://github.com/mendersoftware/REPOSITORY/settings/access
+
+### Enterprise fork
+
+TODO
```

Comment:
Don't forget to fill in this section :)

Reply:
Actually, I don't really know what is special about it (the only thing I remember is for `integration-test-bot`, and I added it in that section already). I was thinking to leave it TODO until we do a new repository with enterprise fork.

Or do you know what else is special about repos with enterprise fork?

Reply:
I don't know anything on top of my head either. I just pointed it out because it's easy to forget about following up inline `TODO`s (at least for me).
Perhaps you can remove this subsection since it's already covered?

Reply:
There are a couple of things I can think of:

1. Change the license to All Rights Reserved
2. Implement closed source license test for Golang files (`CheckMenderCompliance`).

Reply:
And maybe also mention that enterprise Docker repositories are published exclusively to `registry.mender.io`.

Reply:
Thanks. Added